src/lang/en.js

```diff
@@ -1,5 +1,18 @@
 export default {
   en: {
+    accordion: {
```

Comment:
I'd put general at the top deliberately, but fair enough for making it alphabetical